src/pages/edit/ActionsBar.js

```diff
@@ -33,6 +33,10 @@ const saveFailedMessage = i18n.t(
     'Failed to save dashboard. You might be offline or not have access to edit this dashboard.'
 )
 
+const saveFailedCodeExistsMessage = i18n.t(
+    'This code is already being used on another dashboard'
```

Comment:
For consistency:  "Failed to save dashboard. Code is already being used on another dashboard"

Reply:
@cooper-joe ^

Reply:
I don't think "failed to save dashboard" works here, because it's used as an error message underneath a field. The other error (about being offline) is used in an alert bar.

Reply:
@cooper-joe No, this is the alert bar message, I think you mix it up with "Code can't be longer than 50 characters". Unless the intention was to display both of them as an input field error? See screenshots in the PR.

Reply:
Oh yeah, sorry about that. Agreed with @jenniferarnesen! 

I do also think the _unique code_ error should also display as an input field error, as well as an alert bar communicating the failed save.

Reply:
Okay, interesting. That could work, but there are some questions related to that, e.g. for how long should that message remain visible? Same as the alert bar, until the field is edited, semi-permanently until the user leaves the new/edit screen? If the user edits the code the message is no longer correct, unless the user understands that this is only updated after going to the server (not something users should have to consider, imo), as opposed to the max 50 chars check which is instant. As @cooper-joe is on holiday as we speak I'm going to merge the PR for now and revisit this in January.

Reply:
Forgot to update the message, will create a new PR for that.